### PR TITLE
histogram

### PR DESCRIPTION
**Pull request by thfield**
histogram gets data from api call

### CODE REVIEW

**Comment by eayoungs** [comment]
Hey, I'm not getting the server to run. I followed the instructions @ https://help.github.com/articles/checking-out-pull-requests-locally/ and successfully merged locally.
I couldn't find a link in the main index, so I tried this: `http://0.0.0.0:8000/individual/index.html` but it's blank for me.
When I open the file locally, I get: 
image

**Comment by thfield** [comment]
I would try `http://localhost/individual/...` maybe? There are some vague instructions in the main readme, I'm not sure if you knew about those

**Comment by smoningi** [comment]
localhost:8000/individual/index.html should work. Run python -m
SimpleHTTPServer in terminal in the main directory.

**Review by eayoungs — CHANGES_REQUESTED** [review]
Looks like the **data/75rg-imyz.json** doesn't seem to be used in **test.js**. It seems to still be referencing `../data/j2j3-acqj.json`.
I like the page and it seems to run fine at _http://localhost:8000/dashboard/test.html_ but the data doesn't seem to load. Also, the "**Building Type:**" dropdown doesn't seem to be populated.

File: individual/individual.js [hunk 1]
```diff
@@ -1,25 +1,154 @@
 "use strict";
 
 //TODO: CHANGE limit on returned properties in function propertyTypeQuery()
-const DATASOURCE = 'j2j3-acqj' // ? '75rg-imyz'
```
Comment (eayoungs) [inline individual/individual.js:4]: Something is wrong with the data here; does not appear to be loading properly

File: dashboard/dashboard.js [hunk 2]
```diff
@@ -127,7 +127,7 @@ function renderCharts (error, apiData) {
       mapG = mapSVG.append("g").attr("class", "leaflet-zoom-hide");
 
   d3_queue.queue()
-      .defer(d3.json, "../data/j2j3-acqj.json")  /* https://data.sfgov.org/resource/j2j3-acqj.json?$limit=2000 */
+      .defer(d3.json, "../data/j2j3-acqj.json") //this data has already loaded at this point, why is it doing it agian? /* https://data.sfgov.org/resource/j2j3-acqj.json?$limit=2000 */
```
Comment (eayoungs) [inline dashboard/dashboard.js:146]: I looked into this. Not sure but seems to go back some ways, at least to September 2016

File: individual/individual.js (hunk 1 above)
Comment (thfield) [reply individual/individual.js:4]: i think the data source needs to be switched for that soda-js to work properly, not sure why this slipped by.  'j2j3' should be switched for '75rg'.  i think soda-js wants the soda api v2 and 'j2j3' is served through soda api v1.

**Comment by thfield** [comment]
test.js/test.html probably shouldn't have gotten added to the commit, i wouldn't worry about anything going on in them.

File: individual/individual.js (hunk 1 above)
Comment (thfield) [reply individual/individual.js:4]: yup, looks like i switched it in a commit that i had forgotten to push

**Review by eayoungs — APPROVED** [review]

**Pull request merged**